index.html

```diff
@@ -0,0 +1,286 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Development course</title>
+</head>
+
+<body>
+  <div class="mobile-logo-wrap">
+    <div>
+      <a href="index.html">
+        <img src="Images/main-logo.png" alt="developer week global">
+      </a>
+    </div>
+  </div>
+
+  <div class="mobile-menu-wrap">
+    <div class="mobile-menu-button">
+      <a href="#" id="mobile-menu-button">
+        <img src="Images/mobile_menu.png" alt="Mobile menu button">
+      </a>
+    </div>
+
+    <div class="mobile-menu-display display-none" id="mobile-menu-display">
+      <div class="mobile-menu-item align-self-fx-end">
+        <button id="close-mobile-menu">&#10799;</button>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>News</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Sponsor</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Join</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Program</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="about.html">
+          <p>About</p>
+        </a>
+      </div>
+    </div>
+  </div>
+  <!-- Desktop navbar  -->
+  <div class="wrap header-wrap">
+    <div class="main-menu">
+
+      <div class="header-content">
+        <div class="main-logo">
+          <a href="index.html">
+            <img src="Images/main-logo.png" alt="developer week global">
+          </a>
+        </div>
+        <div class="menus">
+          <ul>
+            <li class="menu-item"><a href="about.html">About</a></li>
+            <li class="menu-item"><a href="#">Program</a></li>
+            <li class="menu-item"><a href="#">Join</a></li>
+            <li class="menu-item"><a href="#">Sponsor</a></li>
+            <li class="menu-item"><a href="#">News</a></li>
+          </ul>
+        </div>
+      </div>
+
+    </div>
+  </div>
+
+  <div class="main-wrap wrap">
+    <section>
+      <img id='title-image' src="Images/developer-week-logo.png" alt="developer week february 15-17 sf bay area february 21-23">
+      <div class="message">
+        <p>
+          Each year, 8,000+ developers, engineers, software architects, dev teams, managers, and executives from 70+
+          countries gather for DeveloperWeek (Feb 15–17, San Francisco Bay Area & Feb 21-23, Virtual) to discover the
+          latest in developer technologies, languages, platforms, and tools.
+        </p>
+      </div>
+      <div class="location">
+        <div class="dateinfo">Feb 15-17, 2023</div>
+        <div class="locate">@ Oakland Convention Center</div>
+      </div>
+    </section>
+  </div>
+
+  <div class="wrap main-program-wrap">
+    <section>
+      <h1 class="normal">Main program</h1>
```

Comment:
- [ ] I would recommend that you only use a single `<h1>` tag throughout your HTML file. So use `<h1>` for the main title of your website so that search engines will be able to know what the website is all about  👍🏽

Reply:
Hi @vickymarz , Thanks for your review, my main title is an image of DEVELOPERWEEK event so I don't have any other section where I use that title (only logos), I left only the `<h1 class="normal">Main program</h1>` is that okay?

Reply:
@Sparktan24 , your main title which is an image is not looking like the design template at all, so I will recommend you follow the design guildlines as close as possible. 

Use texts instead of images so that you can style it to look like the design template given 👍🏽 .